wordpress-mobile/WordPress-iOS
# Update Hockey SDK pod to version 5.0.0

Fixes #7841

This PR updates the Hockey SDK to version 5.0.0

To test:
 - Build the app on the Internal or Alpha scheme
 - Check that Hockey SDK is enabled on startup looking at the debug console.

Needs review: @elibud
